src/containers/CampaignList.jsx

```diff
@@ -151,19 +151,28 @@ export class CampaignList extends React.Component {
         style={listItemStyle}
         key={campaign.id}
         primaryText={primaryText}
-        onTouchTap={({
-          currentTarget: {
-            firstElementChild: {
-              firstElementChild: { checked }
-            }
-          }
-        }) => {
+        onTouchTap={event => {
+          event.persist();
```

Comment:
Is event.persist() necessary?

Reply:
Just re-tested: event.persist() is not necessary.
You shall I re-commit or is it easier for you to just handle as an approver?

Please pardon my delay here.

Reply:
@jasmosez commiting the changes and then pushing them will update the PR. Then @lperson can re-review!